rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/toolWindow/log/UnityLogPanelView.kt

```diff
@@ -163,7 +166,7 @@ class UnityLogPanelView(lifetime: Lifetime, project: Project, private val logMod
     @Suppress("SpellCheckingInspection")
     private val listPanel = JPanel(MigLayout("ins 0, gap 0, flowy, novisualpadding, fill", "", "[][min!]")).apply {
         add(JBScrollPane(eventList).apply { horizontalScrollBarPolicy = ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER }, "grow, wmin 0")
-        add(searchTextField, "growx")
+        add(searchTextField, "grow")
```

Comment:
cc @van800

Reply:
I am not an expert in that UI tweaks, but I tried the branch and this change doesn't brake the layout of UnityLogView.

@citizenmatt what this change does?

Reply:
It means the search field will grow both in x and y. So it was stretching to fit horizontally, but didn't grow vertically, so if there was a larger font, the height would be wrong.